i18n/sync_translations.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+if [ "${TRAVIS_EVENT_TYPE}" == "cron" ]
+  then
+    echo "Starting translation sync"
+    set -ev
+    git checkout develop
+    # update translations, and test any updated messages
+    npm run translate
+    npm run translate:update
+    npm run test:messages
+    # stage any changes in the msg directory
+    git add ./msg
+    git commit -m 'update translations from transifex'
```

Comment:
When this gets pushed to develop, I think it will kick off another job, right? It looks like the build published by this cron job will include the changes to `msg`, so that second build will be redundant. I think we should try to make this silent to ci, so that it makes the commit and pushes it, but that one doesn't kick off a new build (the capitalization also made me twitch :)

```suggestion
    git commit -m '[skip ci] Update translations from transifex'
```